package.json

```diff
@@ -35,7 +35,7 @@
     "e2e": "yarn workspaces foreach --all --parallel --verbose run e2e",
     "vrt": "yarn workspaces foreach --all --parallel --verbose run vrt",
     "vrt:docker": "./bin/run-vrt",
-    "rebuild-electron": "./node_modules/.bin/electron-rebuild -f -m ./packages/loot-core",
+    "rebuild-electron": "./node_modules/.bin/electron-rebuild --arch x64 -f -m ./packages/loot-core",
```

Comment:
I'm expecting arm to fail, if it does I may need to figure out how to conditionally call rebuild-electron in the electron-builder package. 

It needs to be in electron-build - package-electron wont work as it doesn't know what arch we're attempting to build. 

I think there's a way to hook into the package and manipulate node_modules